mysql-test/suite/rocksdb/t/index_merge_rocksdb2-master.opt

```diff
@@ -0,0 +1 @@
+--loose-rocksdb_strict_collation_check=off --binlog_format=row --log-bin --transaction-isolation='READ-COMMITTED'
```

Comment:
Actually, I see the problem, index_merge1.inc and possibly others restart mysqld, which wipes/resets any dynamic setting back to default. Thus iso needs to be set here in the .opt file to ensure it survives restarts.